src/floats.jl

```diff
@@ -217,11 +212,20 @@ end
     return pos, code, PosLen(pl.pos, pos - pl.pos), x
 end
 
+@inbounds function handlef(x, f::F) where {F}
```

Comment:
I think this was supposed to be `@inline`

Reply:
Oh man, thank you for catching this! I was investigating a profile the other day and was baffled why there was a dynamic dispatch in this....